### PR TITLE
[release/v2.24] Add k8s 1.28.x to cluster-autoscaler addon manifests

### PR DESCRIPTION
**Pull request by akash-gautam**
**What this PR does / why we need it**:
We are unable to install cluster-autoscaler addon on k8s 1.28.x clusters on KKP 2.24.x
**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind bug
/kind regression

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Add kubernetes 1.28.x support for cluster-autoscaler addon.
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Review by xrstf — APPROVED** [review]
/approve

**Comment by embik** [comment]
@akash-gautam before we merge this please add a release note.

**Comment by embik** [comment]
/hold 

Looks like tests are broken. Probably unrelated to this PR.

**Comment by xrstf** [comment]
/override pre-kubermatic-dualstack-e2e-aws-canal
/override pre-kubermatic-dualstack-e2e-aws-cilium

Both are failing because there is no Rockylinux AMI in AWS eu-west-1, which seemingly can happen (see a previous issue: https://bugs.rockylinux.org/view.php?id=1684).

**Pull request merged**
